tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -38,74 +38,15 @@ class FileConverterTests
         
-        [SetUpFixture]
-        public class WebForms2BlazorTestFixture : AwsRulesBaseTest
-        {
-            public static string TempDir;
-            public static string CopyFolder;
-            public static string DownloadLocation;
-            public static string EShopOnBlazorSolutionPath;
-            public static string EShopOnBlazorSolutionFilePath;
-            public static string EShopLegacyWebFormsProjectPath;
-       
-            [OneTimeSetUp]
-            public void Setup()
-            {
-                Setup(GetType());
-                TempDir = GetTstPath(Path.Combine(new [] { "Projects", "Temp", "W2B" }));
-                DownloadTestProjects();
-       
-                EShopOnBlazorSolutionFilePath = CopySolutionFolderToTemp("eShopOnBlazor.sln", TempDir);
-                EShopOnBlazorSolutionPath = Directory.GetParent(EShopOnBlazorSolutionFilePath).FullName;
-
-                CopyFolder = Directory.GetParent(EShopOnBlazorSolutionPath).FullName;
-                //var tempHelp = Directory.EnumerateFiles(EShopOnBlazorProjectPath, "*.csproj", SearchOption.AllDirectories);
-                
-                EShopLegacyWebFormsProjectPath = Directory
-                    .EnumerateFiles(EShopOnBlazorSolutionPath, "*.csproj", SearchOption.AllDirectories)
-                    .First(filePath =>
-                        filePath.EndsWith("eShopLegacyWebForms.csproj", StringComparison.InvariantCultureIgnoreCase));
-                
-            }
-       
-            private void DownloadTestProjects()
-            {
-                var tempDirectory = Directory.CreateDirectory(TempDir);
-                DownloadLocation = Path.Combine(tempDirectory.FullName, "d");
-       
-                var fileName = Path.Combine(tempDirectory.Parent.FullName, @"TestProjects.zip");
-                Utils.SaveFileFromGitHub(fileName, GithubInfo.TestGithubOwner, GithubInfo.TestGithubRepo, GithubInfo.TestGithubTag);
-                ZipFile.ExtractToDirectory(fileName, DownloadLocation, true);
-            }
-       
-            [OneTimeTearDown]
-            public void Cleanup()
-            {
-                DeleteDir(0);
-            }
-       
-            private void DeleteDir(int retries)
-            {
-                if (retries <= 10)
-                {
-                    try
-                    {
-                        Directory.Delete(TempDir, true);
-                        Directory.Delete(CopyFolder, true);
-                    }
-                    catch (Exception)
-                    {
-                        Thread.Sleep(60000);
-                        DeleteDir(retries + 1);
-                    }
-                }
-            }
-        }
 
         [OneTimeSetUp]
         public void OneTimeSetup()
         {
+            WebForms2BlazorTestFixture = new DownloadTestProjectsFixture();
+            WebForms2BlazorTestFixture.Setup();
+            
```

Comment:
Fixtures shouldn't be instantiated and their setup and teardown methods don't need to be explicitly called since they are automatically executed whenever unit tests are run. If there's any confusion on how these OneTimeSetup/TearDown functions are called, I'd checkout the NUnit docs on TextFixtures and SetupFixtures.

Since the properties we need from the fixture are static, we can reference them statically instead of creating a fixture object.